mathics/builtin/system.py

```diff
@@ -485,6 +485,10 @@ class MemoryAvailable(Builtin):
 
     >> MemoryAvailable[]
      = ...
+
+    The relationship between $SystemMemory, MemoryAvailable, and MemoryInUse:
+    >> $SystemMemory > MemoryAvailable[] > MemoryInUse[]
+     = True
```

Comment:
I suppose they could be equal if all are 0, but that is unthinkable.

Reply:
I like this test!

Reply:
Great - then let's ship.